it/controllers/EstimatesControllerISpec.scala

```diff
@@ -39,14 +42,22 @@ class EstimatesControllerISpec extends ComponentSpecBase {
           And("I wiremock stub a successful Income Source Details response with single Business and Property income")
           IncomeTaxViewChangeStub.stubGetIncomeSourceDetailsResponse(testMtditid)(OK, businessAndPropertyResponse)
 
-          And("I wiremock stub a successful Get Latest Tax Calculation response")
-          IncomeTaxViewChangeStub.stubGetLatestCalculation(testNino, testYear, taxCalculationResponse)
+          And("I stub a successful calculation response for 2017-18")
```

Comment:
is a tax period two years back relevant?

Reply:
in estimate and bills, the user can view the previous tax year calculation detail, so to stub a previous tax year is reasonable for verifying the code has brought back the correct tax year.